_posts/2016-02-18-release-of-bio-formats-cpp-5.1.8.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Release of Bio-Formats-C++ 5.1.8
+intro-blurb: The OME Consortium are pleased to announce the release of Bio-Formats-C++ 5.1.8
+---
+Dear All,
+
+Today we have released the binary builds of Bio-Formats-C++ 5.1.8, together with the ome-cmake-superbuild package used for building Bio-Formats-C++ and its dependencies on Windows and other platforms.  With our current C++ development efforts being devoted to the upcoming 5.2.0 release, this release contains no new features, and is restricted to security updates in our third-party dependencies.  Future 5.1 releases will also be restricted to serious bugfixes and security updates.
+
+See http://downloads.openmicroscopy.org/bio-formats-cpp/5.1.8/ for links to the source releases, documentation and API reference, and binary builds for a number of platforms including Windows, Linux and MacOS X.
```

Comment:
not sure how best to hide this link behind text, @hflynn may want to consider next week